src/components/Calculator.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const [state, setState] = useState({
+    next: null,
+    total: '',
+    operation: null,
+  });
+
+  const handleClick = (e) => {
+    const pressedButton = e.target.innerText;
+    const newState = calculate(state, pressedButton);
+    setState(newState);
+  };
+
+  const { next, total, operation } = state;
+  return (
+
+    <div className="container">
+      <h1>Simple Calculator</h1>
+      <div className="display">
+        <span>{next}</span>
+        <span>{operation}</span>
+        <span>{total}</span>
+      </div>
+      <div>
+        <button value="AC" type="button" onClick={handleClick}>AC</button>
+        <button value="+/-" type="button" onClick={handleClick}>+/-</button>
+        <button value="%" type="button" onClick={handleClick}>%</button>
+        <button value="/" className="orange" onClick={handleClick} type="button"> &#247;</button>
```

Comment:
- [ ] There is a bug 🐛 whenever I try to do a division operation as can be seen in the attached image below 👇 Please try to fix it in order to pass this milestone 🙏 
![image](https://user-images.githubusercontent.com/46839548/198836837-98112b64-9f20-4671-bd31-86d941fc257b.png)